crates/holochain_p2p/src/types/actor.rs

```diff
@@ -143,6 +143,32 @@ impl Default for GetLinksOptions {
     }
 }
 
+#[derive(Debug, Clone)]
+/// Get agent activity from the DHT.
+/// Fields tagged with `[Network]` are network-level controls.
+/// Fields tagged with `[Remote]` are controls that will be forwarded to the
+/// remote agent processing this `GetLinks` request.
+pub struct GetActivityOptions {
+    /// [Network]
+    /// Timeout to await responses for aggregation.
+    /// Set to `None` for a default "best-effort".
+    /// Note - if all requests time-out you will receive an empty result,
+    /// not a timeout error.
+    pub timeout_ms: Option<u64>,
+    /// [Remote]
+    /// Include the activity headers in the response
+    pub include_activity: bool,
```

Comment:
Could you include a little more context on what this option is about and why it's needed? It's not too clear from the code especially since you say it's wrong and is getting fixed later.